modyn/config/schema/pipeline/trigger/performance/performance.py

```diff
@@ -0,0 +1,95 @@
+from typing import Literal
+
+from pydantic import Field, field_validator, model_validator
+
+from modyn.config.schema.base_model import ModynBaseModel
+from modyn.config.schema.pipeline.evaluation.config import EvalDataConfig
+from modyn.config.schema.pipeline.trigger.performance.criterion import (
+    PerformanceTriggerCriterion,
+)
+from modyn.const.types import ForecastingMethod, TriggerEvaluationMode
+
+
+class PerformanceTriggerEvaluationConfig(ModynBaseModel):
+    # max_samples_per_evaluation: int = Field(
+    #     1000,
+    #     description=(
+    #         "The maximum number of samples that should be used for the evaluation. If more are available, "
+    #         "downsampling will be used"
+    #     ),
+    # )  # TODO: currently not yet supported by the evaluator, we use a evaluation dataset that is sufficiently small
+
+    device: str = Field(description="The device the model should be put on.")
+    dataset: EvalDataConfig = Field(description="The dataset on which the model is evaluated.")
+    label_transformer_function: str | None = Field(
+        None, description="Function used to transform the label (tensors of integers)."
+    )
+
+    @field_validator("dataset")
+    @classmethod
+    def validate_metrics(cls, dataset: EvalDataConfig) -> EvalDataConfig:
+        """Assert that we have at least the accuracy metric."""
+        if not any(metric.name == "Accuracy" for metric in dataset.metrics):
+            raise ValueError("The accuracy metric is required for the performance trigger.")
+        return dataset
```

Comment:
Actually why do we need accuracy? If a user wants to trigger based on ROC-AUC (e.g., in recommendation systems), why not? I feel like that is not our beer

Reply:
We don't use the accuracy results but extract the number of misclassifications from them (3rd decision criterion for performance aware triggers)

Reply:
Okay, maybe we can either add a comment here or only enforce this when actually usign this decision criterion, otherwise the changes LGTM

Reply:
** The accuracy information is only used for the misclassification counting, the thresholds-based approaches allow all kinds of user-defined metrics :)